ds.py

```diff
@@ -14,60 +14,65 @@ def create_line_plot(data, **kwargs) -> str:
     return plio.to_json(fig)
 
 
-def create_bar_plot_with_button(data, x, y1, y2, **kwargs) -> str:
+def create_bar_plot_with_button(
+        list_of_series: list,
+        labels: list,
+        default_active=0,
+        xrangeslider=0,
+        **kwargs
+) -> str:
 
     fig = go.Figure()
 
-    fig.add_trace(
-        go.Bar(
-            visible=True,
-            x=data[x],
-            y=data[y1],
-            name=y1,
-            **kwargs
-        )
-    )
-
-    fig.add_trace(
-        go.Bar(
-            visible=False,
-            x=data[x],
-            y=data[y2],
-            name=y2,
-            **kwargs
+    for i, series in enumerate(list_of_series):
+        fig.add_trace(
+            go.Bar(
+                visible=(i == default_active),
+                x=series.index,
+                y=series.values,
+                name=labels[i],
+            )
         )
-    )
 
     # Add button
     fig.update_layout(
         width=1300,
         height=600,
         updatemenus=[
             dict(
-                active=0,
+                active=default_active,
                 type="buttons",
                 direction="left",
-                buttons=list([
-                    dict(
-                        args=[{"visible": [True, False]}],
-                        label=y1,
-                        method="update",
-                    ),
+                buttons=[
                     dict(
-                        args=[{"visible": [False, True]}],
-                        label=y2,
+                        args=[{"visible": [(i == i0) for i0, label in enumerate(labels)]}],
+                        label=label,
                         method="update",
-                    )
-                ]),
+                    ) for i, label in enumerate(labels)
```

Comment:
Og da trenger vi ikke lenger `i`, så det kan bli:

```suggestion
                    ) for label in labels
```